trivial-features.asd

```diff
@@ -39,7 +39,7 @@
      #+clisp      (:file "tf-clisp")
      #+cmu        (:file "tf-cmucl")
      #+cormanlisp (:file "tf-cormanlisp")
-     #+ecl        (:file "tf-ecl")
+     #+(and ecl (not clasp)) (:file "tf-ecl")
```

Comment:
This change is not necessary and should not be in the patch. Clasp does not by default push `:ecl` as a feature.